algolia/search/rendering_content_test.go

```diff
@@ -29,10 +32,15 @@ func TestUnmarshalRenderingContent(t *testing.T) {
 	err := json.Unmarshal([]byte(payload), &r)
 	require.NoError(t, err)
 	require.Equal(t, r.FacetOrdering.Facets.Order, []string{"brand", "size", "color"})
+
 	require.Equal(t, r.FacetOrdering.Values["brand"].Order, []string{"Apple", "Sony", "Samsung"})
 	require.Equal(t, *r.FacetOrdering.Values["brand"].SortRemainingBy, Alpha)
+
 	require.Equal(t, r.FacetOrdering.Values["color"].Order, []string{"red", "green"})
 	require.Nil(t, r.FacetOrdering.Values["color"].SortRemainingBy)
+
+	require.Nil(t, r.FacetOrdering.Values["size"].Order)
+	require.Equal(t, *r.FacetOrdering.Values["size"].SortRemainingBy, Alpha)
```

Comment:
I'm not sure what this line is testing? With or without the change it doesn't matter.

Reply:
Indeed, please disregard this change.